package.json

```diff
@@ -120,52 +121,52 @@
     "npm": ">=1.2"
   },
   "dependencies": {
-    "ambi": "~2.2.0",
-    "backbone": "1.1.2",
-    "bal-util": "~2.5.0",
-    "caterpillar": "~2.0.7",
-    "caterpillar-filter": "~2.0.3",
-    "caterpillar-human": "~2.1.1",
-    "commander": "~2.5.1",
-    "cson": "~1.6.2",
-    "eachr": "~2.0.2",
-    "encoding": "~0.1.11",
-    "envfile": "~1.0.0",
-    "event-emitter-grouped": "~2.4.3",
-    "express": "~3.4.4",
+    "ambi": "^2.2.0",
+    "backbone": "^1.1.2",
+    "bal-util": "^2.5.0",
+    "caterpillar": "^2.0.7",
+    "caterpillar-filter": "^2.0.3",
+    "caterpillar-human": "^2.1.2",
+    "commander": "^2.5.1",
+    "cson": "^1.6.2",
+    "eachr": "^2.0.2",
+    "encoding": "^0.1.11",
+    "envfile": "^1.0.0",
+    "event-emitter-grouped": "^2.4.3",
+    "express": "^3.4.8",
     "extendonclass": "~1.0.1",
-    "extendr": "~2.1.0",
-    "extract-opts": "~2.2.0",
-    "getmac": "~1.0.6",
-    "ignorefs": "~1.0.0",
-    "istextorbinary": "~1.0.0",
-    "jschardet": "~1.1.0",
-    "lazy-require": "~1.0.0",
-    "lodash": "~2.4.1",
-    "mime": "~1.2.9",
-    "progressbar": "~1.0.3",
-    "promptly": "~0.2.0",
-    "query-engine": "~1.5.5",
+    "extendr": "^2.1.0",
+    "extract-opts": "^2.2.0",
+    "getmac": "^1.0.6",
+    "ignorefs": "^1.0.0",
+    "istextorbinary": "^1.0.0",
+    "jschardet": "^1.1.0",
+    "lazy-require": "^1.0.0",
+    "lodash": "^2.4.1",
+    "mime": "^1.2.11",
+    "progressbar": "^1.0.3",
+    "promptly": "^0.2.0",
+    "query-engine": "^1.5.7",
     "rimraf": "^2.2.8",
-    "safefs": "~3.1.2",
-    "safeps": "~2.2.10",
+    "safefs": "^3.1.2",
+    "safeps": "^2.2.12",
     "scandirectory": "^2.5.0",
-    "semver": "~4.1.0",
-    "superagent": "~0.21.0",
-    "taskgroup": "~4.0.5",
-    "typechecker": "~2.0.8",
-    "watchr": "~2.4.12",
-    "yamljs": "~0.2.1"
+    "semver": "^4.1.1",
+    "superagent": "^0.21.0",
+    "taskgroup": "^4.0.5",
+    "typechecker": "^2.0.8",
+    "watchr": "^2.4.12",
+    "yamljs": "^0.2.1"
   },
   "optionalDependencies": {
     "longjohn": "~0.2.4"
   },
   "devDependencies": {
-    "coffee-script": "~1.8.0",
-    "chai": "~1.10.0",
-    "joe": "~1.5.0",
-    "joe-reporter-console": "~1.2.1",
-    "projectz": "~0.3.17"
+    "chai": "^1.10.0",
+    "coffee-script": "^1.8.0",
+    "joe": "^1.5.0",
+    "joe-reporter-console": "^1.2.1",
+    "projectz": "^0.3.17"
```

Comment:
Myself I am not sure of the difference, usually I use whatever `--save` gives me...